src/node_buffer.cc

```diff
@@ -49,6 +49,10 @@
   size_t length = end - start;
 
 namespace node {
+
+// if true, all Buffer and SlowBuffer instances will automatically zero-fill
+bool zero_fill_all_buffers = false;
+
```

Comment:
Think we could just make a:
```cpp
void* buffer_malloc(length) {
  return zero_fill_all_buffers ? calloc(length, 1) : malloc(length);
}
```
To reduce the number of conditionals in the rest of the code?